guides/user/canary/best-practices/index.md

```diff
@@ -16,6 +16,112 @@ critical metric fails, but the rest pass, the group gets a passing score overall
 You can put a critical metric in a group of only one to ensure that if it fails,
 the whole group fails every time.
 
+## Compare canary against baseline, not against production
+
+You might be tempted to compare the canary deployment against your current
+production deployment. Instead always compare the canary against an equivalent
+baseline, with both deployed together.
+
+The baseline uses the same version and configuration that is currently running
+in production, but is otherwise identical to the canary:
+
+* Same time of deployment
+* Same size of deployment
+* Both receive the same amount of traffic
+
+Doing this, you control for version and configuration only, and you reduce
+factors that could affect the analysis, like the cache warmup time, the heap
+size, and so on.
+
+## Run the canary for a long-enough time
+
+You need at least 50 data points for the statistical analysis to produce
```

Comment:
Does a reader understand at this point what a "data point" is?

Reply:
Eh, I changed this a bit, but I don't think this page is the place to ensure they understand what they're getting into with metrics. That is, if they're going to get metrics from a provider and use them, I'm going to assume (on this page at least) that they understand what a time series is, or they'll realized that they're going to need to.